devDocs/startupShutdown.md

```diff
@@ -43,41 +57,35 @@
     1. All wx windows are closed
     1. Now that windows are closed, a new NVDA instance is started if requested
 
-### Out of process: `WM_QUIT`
-* [a windows message](https://docs.microsoft.com/en-us/windows/win32/winmsg/wm-quit) which can be sent across processes which will force the main loop to exit and close most NVDA windows.
+### When exiting from `WM_QUIT`
+* [A Windows Message](https://docs.microsoft.com/en-us/windows/win32/winmsg/wm-quit) received from an external process, such another NVDA process.
+* NVDA accepts [WM_QUIT](#When-exiting-from-WM_QUIT) messages from other processes and creates a [named window](https://docs.microsoft.com/en-us/windows/win32/learnwin32/creating-a-window#creating-the-window) that can be discovered.
+* Will force the main loop to exit and close most wx Windows.
 * We subsequently run `triggerNVDAExit` to ensure that clean up code isn't missed, and pump the queue to execute it.
 * Changing to using a custom message, which would allow a custom handling (eg just `triggerNVDAExit`), requires compatibility with messaging older NVDA versions that are only aware of `WM_QUIT`.
 
-### Out of process: `wx.EVT_END_SESSION`
-- This is a [wxCloseEvent](https://docs.wxwidgets.org/3.0/classwx_close_event.html) triggered by a Windows session ending.
-- On `wx.EVT_END_SESSION`, we save the config and play the exit sound.
-- Other actions are not performed as we have limited time to perform an action for this event. With NVDA being killed off very last, Windows may include NVDA in the Block shutdown dialog, but the user won't be able to read it if we are shutting down.
-
-## Technical startup and exit notes
+### When exiting from `wx.EVT_END_SESSION`
+* This is a [wxCloseEvent](https://docs.wxwidgets.org/3.0/classwx_close_event.html) triggered by a Windows session ending.
+* On `wx.EVT_END_SESSION`, we save the config and play the exit sound.
+* Other actions are not performed as we have limited time to perform an action for this event. With NVDA being killed off very last, Windows may include NVDA in the Block shutdown dialog, but the user won't be able to read it if we are shutting down.

```

Comment:
Could you please clarify the final sentence here?
- Is NVDA expected to run in the blocked shutdown dialog?
- How does NVDA ensure that it is still running, or ensure that it is killed off last?

Also how much time does NVDA have?

Reply:
>Is NVDA expected to run in the blocked shutdown dialog?

It tends to, until it is killed off last

>Also how much time does NVDA have?

I think this varies on the system

>How does NVDA ensure that it is still running, or ensure that it is killed off last?

I'm not sure here - @michaelDCurran?

Reply:
```suggestion
* Other actions are not performed as we have limited time to perform an action for this event.
NVDA is expected to run as long as possible during the sign out process.
With NVDA being killed off very last, Windows may include NVDA in the Block shutdown dialog, but the user won't be able to read it if we are shutting down.
```

Reply:
Rather than "we", prefer "NVDA" or "the process"

> It tends to, until it is killed off last

What makes it last? Is there code in NVDA that controls this?